Makefile

```diff
@@ -49,6 +49,7 @@ resources = deploy/namespace.yaml deploy/service_account.yaml deploy/role.yaml d
 all: check handler
 
 check: format vet whitespace-check
+	hack/check-go-version.sh
```

Comment:
seems like a very good idea, but unrelated to the purpose of this PR. Please split to another PR with a proper title and purpose.